### ISSUE TITLE
name 'MigrationStatus' is not defined

### ISSUE THREAD

**Issue by DLu** [awslabs/ros2-migration-tools#1]
Command line invocation: `python3 ../ros2-migration-tools/ros2_migration/ros2_migration_tool.py check-deps nav_core2`
Python Version: 3.5.2
Tool version: Checked out from source, master (as of 10/17/2018)

Error: 
```
Traceback (most recent call last):
  File "../ros2-migration-tools/ros2_migration/ros2_migration_tool.py", line 107, in <module>
    cli()
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.5/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "../ros2-migration-tools/ros2_migration/ros2_migration_tool.py", line 102, in check_deps
    from effort_estimation import effort_estimation
  File "/home/dlu/ROS2/ros2-migration-tools/ros2_migration/effort_estimation/effort_estimation.py", line 20, in <module>
    from .ros2_release_checker import get_package_status
  File "/home/dlu/ROS2/ros2-migration-tools/ros2_migration/effort_estimation/ros2_release_checker.py", line 33, in <module>
    class MigrationStatus():
  File "/home/dlu/ROS2/ros2-migration-tools/ros2_migration/effort_estimation/ros2_release_checker.py", line 39, in MigrationStatus
    def __init__(self, status=MigrationStatus.unknown, detail=""):
NameError: name 'MigrationStatus' is not defined

```

Entirely possible I'm doing something dumb wrong.

**Comment by ryanewel** [comment]
That was a bug. Thanks for catching it! It should be fixed now.

**Comment by DLu** [comment]
Closed via 10d68f0b4c625ad0376e0f4b0fadd695e697b7dc
